common/notification/actions.ts

```diff
@@ -314,6 +315,23 @@ const _notificationActions = {
             redirectTo: '/start',
         },
     },
+    'student-cancel-appointment-group': {
```

Comment:
most of the notification actions are with underscores and I think we should keep the style